RubeesEat/Model/IBillRepository.cs

```diff
@@ -4,4 +4,6 @@ public interface IBillRepository
 {
     IReadOnlyList<Bill> GetAll();
     void Add(Bill bill);
+    IReadOnlyList<Bill> GetAllForUser(Guid userGuid);
+    IReadOnlyList<BalanceChange> GetRecentBalanceChanges(Guid userGuid, int amount);
```

Comment:
Just noticed, we should be using the domain objects not the IDs here. So please change the two methods to take a `Person` instead.